lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -572,7 +572,7 @@ def main():
         http_timeout=dict(type='int'),
         load_path=dict(type='path'),
         name=dict(type='str', required=True),
-        nocache=dict(type='str', default=False),
+        nocache=dict(type='bool', default=False),
```

Comment:
Hi,
In the `DOCUMENTATION` block please use `type: bool` and NO `choices:`

http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#documentation-block

If you could tidy up all instances of that in the `DOCUMENTATION` anywhere `dict(type='bool'` is used that would be great.